discovery/zookeeper/zookeeper_test.go

```diff
@@ -0,0 +1,454 @@
+package zookeeper
+
+import (
+	"github.com/joyent/containerpilot/discovery"
+	"github.com/samuel/go-zookeeper/zk"
+	"testing"
+	"time"
+)
+
+// Factories, utilities
+func zkConnection() *zk.Conn {
+	c, _, _ := zk.Connect([]string{"127.0.0.1"}, time.Second)
```

Comment:
Yes, I’ll work on this part once after sorting out the first phase of the review.